HabeebAwawda/task.py

```diff
@@ -0,0 +1,42 @@
+class Task:
+    # one
+
+    task_id = 0
+    title = ""
+    description = ""
+    due_date = ""
+    status = ""
```

Comment:
Using empty strings as default values for title, description, due_date, and status is fine, but consider using Optional[str] with None for better type safety.